### ISSUE TITLE
Failed to submit 

### ISSUE THREAD

**Issue by plitter** [ebekker/ACMESharp#201]
PS C:\Users\uniadmin> Submit-ACMECertificate <alias>
Submit-ACMECertificate : The remote name could not be resolved: 'acme-v01.api.letsencrypt.org'
At line:1 char:1
+ Submit-ACMECertificate <alias>
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Submit-ACMECertificate], WebException
    + FullyQualifiedErrorId : System.Net.WebException,ACMESharp.POSH.SubmitCertificate

**Comment by ebekker** [comment]
Are you saying you got the unresolved DNS name error when you specified an invalid "Identifier Alias"?

**Comment by plitter** [comment]
No, the alias was correct, or it didn't complain when I made it at least(there was an _ in it, but not sure if that's the error). There was some errors with the connection, so maybe its the case that connection was down when I tried to run submit and it couldn't resolve. Thanks for fast response :)

**Comment by ebekker** [comment]
Cool, yeah it sounds like this was just a case of the DNS name was temporarily not resolving for some reason.

**Comment by plitter** [comment]
I guess this is a small usecase, but I would maybe add it to the wiki or allow for asset file to resubmit the request that it made or make the program not "save" that it has been sent, so that you can resubmit.

**Comment by ebekker** [comment]
Good ideas -- I think the underlying problem is a bug in the error handling logic and the code not cleaning up after itself properly or testing for this corner case.
